src/diffTool/rulesProcessor.ts

```diff
@@ -63,24 +63,22 @@ function getRules(rulesPath: string): RuleProperties[] {
   }
   let rules: RuleProperties[];
   try {
-    rules = JSON.parse(fs.readFileSync(rulesPath, "UTF-8"));
+    rules = fs.readJSONSync(rulesPath);
   } catch (error) {
     error.message = `Error parsing the rules file '${rulesPath}': ${error.message}`;
     throw error;
   }
```

Comment:
```suggestion
const rules: RuleProperties[] = fs.readJSONSync(rulesPath);
```
The modified error message doesn't provide any new information.

Reply:
readJSONSync does not provide user friendly message in all the cases like empty string